mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMessageManager.java

```diff
@@ -356,7 +356,7 @@ public ComposedMessageId appendMessage(InputStream msgIn, Date internalDate, fin
                 // component and not the parser itself
                 // https://issues.apache.org/jira/browse/IMAP-122
                 final MimeTokenStream parser = getParser(bIn);
-                final HeaderImpl header = readHeader(parser);
+                readHeader(parser);
```

Comment:
isn't it a bug?

Reply:
Probably a side effect...

Reply:
I did not take much time to check it, but it seems the stream is just consumed. But I'm happy it raises concern, because I was a little worried about it.

Reply:
do we open a ticket for that?